appengine/standard/migration/ndb/redis_cache/main.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
nit: please bump the copyright year to 2020

Reply:
Fixed.